src/upload_data.py

```diff
@@ -0,0 +1,23 @@
+import boto3
+import requests
+import argparse
+import os
+
+cwd = os.getcwd()
+
+data_permanent_url = 'http://s3.us-east-2.amazonaws.com/nw-carsonchen-acedata/atp_data.csv'
```

Comment:
data_permanent_url and file_name could be moved to the config.py!

Reply:
Good suggestion. I moved it.